### PR TITLE
zint: add page 

### PR DESCRIPTION
**Pull request by lsomm**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known): 2.13.0**

First time contributing, let me know what to improve! 🙃

### CODE REVIEW

File: pages/common/zint.md
Comment (spageektti) [inline pages/common/zint.md:8]:
```suggestion
`zint --data {{your_data_in_utf8}} --output {{path/to/file}}`
```
This doesn't require a full path.

File: pages/common/zint.md
Comment (spageektti) [inline pages/common/zint.md:12]:
```suggestion
`zint --barcode {{barcode_type}} --data {{your_data_in_utf8}} --output {{path/to/file}}`
```

**Review by spageektti — COMMENTED** [review]
Hi. Thanks for your contribution. I left a suggestion.

**Comment by lsomm** [comment]
This seems reasonable, @spageektti, thanks for looking at it! I have applied them as you suggested.

File: pages/common/zint.md
Comment (acuteenvy) [inline pages/common/zint.md:3]: Please avoid specifying that the program is used on the command-line.
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#wording
```suggestion
> Generate barcodes and QR codes.
```

File: pages/common/zint.md
Comment (acuteenvy) [inline pages/common/zint.md:12]: The official manual puts the text inside quotes, since it frequently contains spaces.
I think we could use that here too.
```suggestion
`zint --data "{{UTF-8 data}}" --output {{path/to/file}}`

- Specify a barcode type (list all possible values with `zint --types`) for generation:

`zint --barcode {{barcode_type}} --data "{{UTF-8 data}}" --output {{path/to/file}}`
```

**Review by acuteenvy — APPROVED** [review]

File: pages/common/zint.md
Comment (kbdharun) [inline pages/common/zint.md:10]:
```suggestion
- Specify a barcode type for generation:
```

Since, this page only has 3 examples, we can make this as a separate example.

**Review by kbdharun — APPROVED** [review]
Welcome to tldr and thanks for your contribution.

LGTM, after these suggestions are addressed.

File: pages/common/zint.md
Comment (lsomm) [reply pages/common/zint.md]: Implemented in 0c38fda and d25bfc3.

**Comment by lsomm** [comment]
Thank you all for your suggestions! I have implemented them all and slightly rephrased the wording to better differentiate on which commands generate actual barcodes (the first one does by default) and which generate any type of code (bar+QR+...). I hope those are fine, if not, I can revert them.

**Comment by lsomm** [comment]
Out of interest, how long will it take for those changes to show up in the updated python release?

**Comment by kbdharun** [comment]
> Out of curiosity, how long will it take for those changes to show up in the updated python release?

If you are using our Python client (which used git primarily along with caching), it would show up immediately once this PR gets merged.

For other clients, the cache would either update automatically when you type in the name of newly added page, or you can update it manually by passing the `-u` (update) flag in your client.

**Pull request merged**
